pkg/oc/cli/admin/migrate/storage/storage.go

```diff
@@ -267,13 +263,11 @@ func (o *MigrateAPIStorageOptions) Complete(f kcmdutil.Factory, c *cobra.Command
 	clientConfigCopy.Burst = 99999
 	clientConfigCopy.QPS = 99999
 
-	client, err := dynamic.NewForConfig(clientConfigCopy)
+	o.client, err = dynamic.NewForConfig(clientConfigCopy)
```

Comment:
Ew.

Reply:
?
We were setting the client a few lines below. Figured it was cleaner to assign it directly

Reply:
I am just not a fan of that pattern.

Reply:
@enj what's wrong with it? Why would you prefer setting a temporary variable and then assign it to a struct member instead of direct assignment? I'm curious.

Reply:
Feels weird to me to set a struct member before first checking the error.  But it is really just a preference.